docsk8s/administering/ls-k8s-stack-monitoring.asciidoc

```diff
@@ -1,14 +1,122 @@
 [[ls-k8s-stack-monitoring]]
-=== Set up stack monitoring
+=== Stack Monitoring with ECK
```

Comment:
I wonder if this should be a sub-heading, rather than the main heading, with the main heading being `Stack Monitoring`

Reply:
👍 would be helpful to distinguish prerequisite